.pfnci/matrix.yaml

```diff
@@ -230,32 +230,6 @@
   env:CUPY_ACCELERATORS: "cutensor,cub"
   test: "slow"
 
-###############################################################################
-# Doctest
-###############################################################################
-
-# Doctest | Linux
-# Notes:
-# - Library versions may be overridden by docs/requirements.txt
-# - Accelerators are disabled as doctests expects it (#5891)
-- project: "cupy.linux.cuda-doctest"
-  target: "cuda-doctest"
-  system: "linux"
-  os: "ubuntu:20.04"
-  cuda: "11.4"
-  rocm: null
-  nccl: "2.10"
-  cutensor: "1.3"
-  cusparselt: "0.1.0"
-  cudnn: "8.2"
-  python: "3.9"
-  numpy: "1.21"
```

Comment:
python 3.9 -> 3.7, numpy 1.21 -> 1.20 after the change.  Could you confirm these are what you want?

Reply:
I think running examples/doctests with minimum supported python version is actually better 😃  Added comment.